example/lambdas/db-provision-user-database/package.json

```diff
@@ -23,6 +23,7 @@
   },
   "dependencies": {
     "@cumulus/common": "2.0.1",
+    "@cumulus/db": "^2.0.1-alpha.1",
```

Comment:
I think we should be pinning to exact versions?

Reply:
f0f90665a